src/apps/paths/PathViewer.tsx

```diff
@@ -17,6 +17,7 @@ import {
   useState
 } from 'react';
 import { TinaMarkdown } from 'tinacms/dist/rich-text';
+import CoreDataMedia from '@components/CoreDataMedia';
```

Comment:
Could we rename this something like `Media` or `MediaInsert` to be consistent with the `Place` component?